app/code/ElTrubetskaia/Lesson5/view/frontend/templates/additionalpopupcustomer.phtml

```diff
@@ -0,0 +1,406 @@
+<?php
+
+use Magento\Customer\Helper\Address;
+use Magento\Customer\Block\Widget\Fax;
+use Magento\Customer\Block\Widget\Telephone;
+use Magento\Customer\Block\Widget\Company;
+use Magento\Customer\Block\Widget\Gender;
+use Magento\Customer\Block\Widget\Taxvat;
+use Magento\Customer\Block\Widget\Dob;
+use Magento\Customer\Block\Widget\Name;
+
+/**
+  * @var \Magento\Customer\Block\Form\Register $block
+ */
+?>
+<h3>
+    <a href="javascript:void(0)"
+       id="registration-for-dealer"
+       data-mage-init='{"modalRegistrationForm": {}}'
+    >
+        <?php echo $block->escapeHtml(__('Registration for dealer')) ?>
+    </a>
+</h3>
+
+<div id="registration-for-dealer-modal" style="display:none;">
+    <div id="registration-for-dealer-modal-content">
+        <form class="form create account form-create-account"
+              action="<?php echo $block->escapeUrl($block->getPostActionUrl()) ?>"
```

Comment:
Use short `<?=` syntax instead of `<?php echo`. Besides, native Magento templates use it.